src/ontology/mondo-edit.obo

```diff
@@ -444446,16 +444418,29 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 [Term]
 id: MONDO:0020772
 name: epilepsy, juvenile absence, susceptibility to, 1
+subset: gard_rare {source="GARD:2162"}
+subset: nord_rare {source="MONDO:NORD"}
+subset: ordo_disease {source="Orphanet:1941"}
+subset: orphanet_rare {source="Orphanet:1941"}
+subset: rare
 synonym: "EJA1" EXACT ABBREVIATION [OMIM:607631]
 synonym: "EPILEPSY, JUVENILE ABSENCE, SUSCEPTIBILITY TO, 1" EXACT [OMIM:607631]
+synonym: "epilepsy, juvenile absence, susceptibility to, type 1" EXACT [MONDORULE:1, OMIM:607631]
 synonym: "JAE1" EXACT ABBREVIATION [OMIM:607631]
+synonym: "susceptibility to juvenile absence epilepsy 1" EXACT [OMIM:607631]
 xref: DOID:0111324 {source="MONDO:equivalentTo"}
 xref: OMIM:607631 {source="MONDO:equivalentTo"}
 xref: Orphanet:1941 {source="OMIM:607631"}
-xref: UMLS:C2750892 {source="OMIM:607631"}
+xref: UMLS:C2750892 {source="MONDO:notFoundInDiseaseSubset", source="OMIM:607631"}
 is_a: MONDO:0020573 {source="OMIM:607631"} ! inherited disease susceptibility
-relationship: excluded_subClassOf MONDO:0011876 {source="OMIM:607631", source="https://orcid.org/0000-0001-5208-3432"} ! juvenile absence epilepsy
-relationship: predisposes_towards MONDO:0011876 {source="OMIM:607631"} ! juvenile absence epilepsy
+is_a: MONDO:1030001 ! epilepsy, juvenile absence, susceptibility to
+intersection_of: MONDO:0020573 ! inherited disease susceptibility
+intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/16406 ! EFHC1
+intersection_of: predisposes_towards MONDO:0800453 ! juvenile absence epilepsy
+relationship: excluded_subClassOf MONDO:0100030 {source="https://orcid.org/0000-0001-8486-0558"} ! adolescent/adult-onset epilepsy syndrome
+relationship: excluded_subClassOf MONDO:0800453 {source="OMIM:607631", source="https://orcid.org/0000-0001-5208-3432"} ! juvenile absence epilepsy
+relationship: has_characteristic MONDO:0021152 {source="OMIMPS:607631"} ! inherited
```

Comment:
The source should be OMIM:607631 and not OMIMPS:607631

Reply:
fixed with 12b2ae302